HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1333,52 +1378,89 @@ def get_external_file
   end
 
   def set_vacancy
-    return unless @tmp_schedules.keys.include? 'vacancy'
-    return if @tmp_schedules['vacancy'].all? { |i| i == 0 }
+    return unless @tmp_schedules.keys.include? ColumnVacancy
+    return if @tmp_schedules[ColumnVacancy].all? { |i| i == 0 }
 
     col_names = SchedulesFile.ColumnNames
 
-    @tmp_schedules[col_names.keys[0]].each_with_index do |ts, i|
-      col_names.keys.each do |col_name|
-        next if col_names[col_name].nil?
-        next unless col_names[col_name] # skip those unaffected by vacancy
+    @tmp_schedules[col_names[0]].each_with_index do |ts, i|
+      col_names.each do |col_name|
+        next unless affected_by_vacancy[col_name] # skip those unaffected by vacancy
 
-        @tmp_schedules[col_name][i] *= (1.0 - @tmp_schedules['vacancy'][i])
+        @tmp_schedules[col_name][i] *= (1.0 - @tmp_schedules[ColumnVacancy][i])
       end
     end
   end
 
   def self.ColumnNames
-    # col_name => affected_by_vacancy
-    return {
-      ColumnOccupants => true,
-      ColumnLightingInterior => true,
-      ColumnLightingExterior => true,
-      ColumnLightingGarage => true,
-      ColumnLightingExteriorHoliday => true,
-      ColumnCookingRange => true,
-      ColumnRefrigerator => false,
-      ColumnExtraRefrigerator => false,
-      ColumnFreezer => false,
-      ColumnDishwasher => true,
-      ColumnClothesWasher => true,
-      ColumnClothesDryer => true,
-      ColumnCeilingFan => true,
-      ColumnPlugLoadsOther => true,
-      ColumnPlugLoadsTV => true,
-      ColumnPlugLoadsVehicle => true,
-      ColumnPlugLoadsWellPump => true,
-      ColumnFuelLoadsGrill => true,
-      ColumnFuelLoadsLighting => true,
-      ColumnFuelLoadsFireplace => true,
-      ColumnPoolPump => false,
-      ColumnPoolHeater => false,
-      ColumnHotTubPump => false,
-      ColumnHotTubHeater => false,
-      ColumnHotWaterDishwasher => true,
-      ColumnHotWaterClothesWasher => true,
-      ColumnHotWaterFixtures => true,
-      ColumnVacancy => nil,
-    }
+    return SchedulesFile.OccupancyColumnNames
```

Comment:
I don't understand... `ColumnNames` is the same as `OccupancyColumnNames`? What's the point?

Reply:
It's the setup for the `wh-geb` branch; eventually they won't be the same.